ClearBank.ApiTest/ClearBank.ApiTest/Steps/DeckOfCardsSteps.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ClearBank.ApiTest.Models;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using RestSharp;
+using System;
+using TechTalk.SpecFlow;
+
+namespace ClearBank.ApiTest
+{
+    [Binding]
+    public class DeckOfCardsSteps
+    {
+        private RestClient restClient;
+        private RestRequest restRequest;
+        private String deckId;
+        private String remaining;
+     
+
+        [Given(@"I create a new deck of cards")]
+        public void GivenICreateANewDeckOfCards()
+        {
+            restClient = new RestClient("https://deckofcardsapi.com/api/");
```

Comment:
Things like default uri's etc should be stored in a config and called from there. That way you only need to update 1 place if the uri changes.